### PR TITLE
[WhiteHouse.gov.xml] remove already preloaded

### PR DESCRIPTION
**Pull request by nextPrime**
https://whitehouse.gov has been preloaded in Chrome's HSTS preload list for quite a long time. So there is no need to keep this ruleset in HTTPS Everywhere. (Related: #2163, #3454)

[1] https://hstspreload.appspot.com/?domain=whitehouse.gov

### CODE REVIEW

**Comment by nextPrime** [comment]
https://www.servicesonline.opm.gov/ returns 200 when tested on my computer, but the check keeps telling me:
`ERROR src/chrome/content/rules/OPM.gov.xml: Non-2xx HTTP code: http://www.servicesonline.opm.gov/ (200) => https://www.servicesonline.opm.gov/ (403)`

How should this be fixed?

**Comment by Bisaloo** [comment]
When tested with `curl` or `wget`, it returns 403 (even with http for me). That is why the test is failing.

I can confirm it works for me with firefox though.

File: src/chrome/content/rules/OPM.gov.xml [hunk 1]
```diff
@@ -116,15 +115,18 @@
...
 
 
 	<securecookie host="." name="." />
 
 
-	<!-- could not resolve host -->
+	<!-- could not resolve host 
 	<rule from="^http://servicesonline\.opm\.gov/"
-		to="https://www.servicesonline.opm.gov/" />
+		to="https://www.servicesonline.opm.gov/" />-->
```
Comment (Hainish) [inline src/chrome/content/rules/OPM.gov.xml]: This seems no longer to be the case.  I'm able to access https://www.servicesonline.opm.gov/ fine.

**Comment by Hainish** [comment]
Other than my 1 comment, lgtm

File: src/chrome/content/rules/OPM.gov.xml (hunk 1 above)
Comment (Hainish) [inline src/chrome/content/rules/OPM.gov.xml]: This is one of those  weird cases where the response code is altered by the server based on the User-Agent:

```shell
legind@EFF-420 ~/workspace/https-everywhere (nextPrime-rm_whitehouse) $ curl -sD /dev/stdout -o /dev/null 'https://www.servicesonline.opm.gov/' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36' | head -n 1
HTTP/1.1 200 OK
legind@EFF-420 ~/workspace/https-everywhere (nextPrime-rm_whitehouse) $ curl -sD /dev/stdout -o /dev/null 'https://www.servicesonline.opm.gov/' | head -n 1HTTP/1.1 403 Forbidden
```

I say leave this unchanged, we can merge anyway if this is the only problematic fetch url.

**Comment by nextPrime** [comment]
Thanks for your comment! I have updated the patch to keep that rule.

File: src/chrome/content/rules/OPM.gov.xml [hunk 2]
```diff
@@ -116,7 +115,10 @@
 	<!--	Complications:
 				-->
 	<target host=    "www.leadership.opm.gov" />
+	
+	<!-- Unable to connect via both HTTP and HTTPS
 	<target host=    "servicesonline.opm.gov" />
+	-->
```
Comment (Hainish) [inline src/chrome/content/rules/OPM.gov.xml]: We should keep this target how it was as well

**Comment by Hainish** [comment]
@nextPrime I've also commented on https://github.com/EFForg/https-everywhere/pull/5119/files#diff-2423f073984b3d2930ef07f4a22c0b90R121, when you revert that I can re-run tests and merge

**Comment by nextPrime** [comment]
@Hainish Done!

**Comment by Hainish** [comment]
Ok, once the tests run and we see only that 1 failure I'll merge

**Pull request merged**